pom.xml

```diff
@@ -65,6 +66,11 @@
             <artifactId>jasperreports-fonts</artifactId>
             <version>${jasperrport.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Why do we need to include bouncy castle?

Reply:
After further investigation, this should be done once the following feature is implemented and integrated on the main branch of this repo:
* https://github.com/systelab/java-report-generator/issues/25
* https://github.com/systelab/java-report-generator/tree/25-get-password-protected-reports

I would suggest not including it until then.